server/portal/templates/includes/header.html

```diff
@@ -26,7 +26,7 @@
       {# NOTE: As of 2020-12, search is only available with a Portal #}
       {% include "includes/nav_search.html" with className="form-inline  ml-auto" only %}
       {# FAQ: Unlike CMS and User Guide, we must pass `user` #}
-      {% include "includes/nav_portal.html" with className="navbar-nav" user=user only %}
+      {% include "includes/nav_portal.html" with className="navbar-nav" user=user settings=settings only %}
```

Comment:
Instead of passing the whole of `settings` to the template renderer, could we pass just the setting we need?

Reply:
I asked the same question recently. (My context was a Core-CMS portal nav to serve tup-ui and Core-Portal.)

[I abandoned my effort and did not answer that question.](https://github.com/TACC/tup-ui/pull/58#discussion_r1002044956)

> 2. Should we pass many context variables* or a few context objects†?
> ...
>
> \* i.e. `user`, `url_thing_a`, `url_thing_b`, `hide_thing_a`, [`settings_hide_thing_a`, ] et cetera
> † i.e. `user`, `urls`, `hide`, [`settings`]

Reply:
Yeah good point. It probably doesn't add too much overhead to pass the whole settings object as an example. I suppose if it's already wrapped into a big object, it should be fine

Reply:
FWIW, I'd done a similar investigation and came to the same conclusion as Wes.

Reply:
Discussion about parameterization of customizable portal nav menu—I support this, but have too many unknowns—can continue at https://github.com/TACC/tup-ui/pull/58#discussion_r1003692989.

Reply:
New ticket [FP-1913](https://jira.tacc.utexas.edu/browse/FP-1913).